.github/workflows/test-runner-nightly-integration.yml

```diff
@@ -56,6 +56,7 @@ jobs:
           cd workspace-manager-clienttests
           echo "Test Runner config"
           ./render-config.sh ${{ steps.vault-token-step.outputs.vault-token }}
+          ./render-k8s-config.sh wsmtest ${{ steps.vault-token-step.outputs.vault-token }}
           echo "Running test suite"
           ./gradlew runTest --args="suites/BasicIntegration.json /tmp/TestRunnerResults"
```

Comment:
I think it's helpful to always override the server specification with the env var, so it's clear what server we're running against and so that the behavior doesn't change unexpectedly if someone changes the default server in the test suite file.

Reply:
Added server specification env var to all GH action workflows.